### PR TITLE
[KHRGA-79] Updated DA sampled image accessor

### PR DESCRIPTION
**Pull request by imvucic**
Issue: KHRGA-79

### CODE REVIEW

File: source/iface/sampled-image-accessor.rst [hunk 1]
```diff
@@ -2,20 +2,220 @@
...
 
 .. _host_sampled_image_accessor:
 
 =======================================
 ``sycl::host_sampled_image_accessor``
 =======================================
+
+::
+
+  template <typename DataT, int Dimensions> class host_sampled_image_accessor;
```
Comment (mkinsner) [inline source/iface/sampled-image-accessor.rst:72]:
```suggestion
  template <typename DataT, int Dimensions>
  class host_sampled_image_accessor;
```

File: source/iface/sampled-image-accessor.rst [hunk 2]
```diff
@@ -2,20 +2,220 @@
...
+
+Returns the number of elements of the underlying
+``sampled_image`` that this accessor is accessing.
+
+``read``
+========
+
+::
+
+  template <typename CoordT> DataT read(const CoordT& coords) const
```
Comment (mkinsner) [inline source/iface/sampled-image-accessor.rst:109]:
```suggestion
  template <typename CoordT> DataT
  read(const CoordT& coords) const
```

For this reference, it's probably better to split lines like suggested here, for readability.

File: source/iface/sampled-image-accessor.rst (hunk 1 above)
Comment (imvucic) [reply source/iface/sampled-image-accessor.rst]: Fixed with https://github.com/KhronosGroup/SYCL_Reference/pull/106/commits/0fd5efe01092231cbb851420c283210d0f1c1cb0

File: source/iface/sampled-image-accessor.rst (hunk 2 above)
Comment (imvucic) [reply source/iface/sampled-image-accessor.rst]: Fixed with https://github.com/KhronosGroup/SYCL_Reference/pull/106/commits/0fd5efe01092231cbb851420c283210d0f1c1cb0

**Review by mkinsner — APPROVED** [review]

**Pull request merged**